### ISSUE TITLE
Add default options to documentation?

### ISSUE THREAD

**Issue by bewuethr** [adrienverge/yamllint#290]
In the docs, the [default configuration](https://yamllint.readthedocs.io/en/stable/configuration.html#default-configuration) shows which rules are enabled, and under [Rules](https://yamllint.readthedocs.io/en/stable/rules.html), I can look up the rules and their options.

However, unless I've just not seen it, there doesn't seem to be documentation for the default options. For example, for [indentation](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.indentation), the default can be seen in the source:

https://github.com/adrienverge/yamllint/blob/8b758d4e7e1bb4cf32bb5c181cefa48247e2d1c3/yamllint/rules/indentation.py#L204-L206

Wouldn't it be useful to make these part of the documentation? Would a PR for this be welcome? I'm thinking something like

---

## indentation

Use this rule to control the indentation.

### Options

- `spaces` defines the indentation width, in spaces. Set either to an integer (e.g. `2` or `4`, representing the number of spaces in an indentation level) or to `consistent` to allow any number, as long as it remains the same within the file.
- `indent-sequences` defines whether block sequences should be indented or not [...]
- `check-multi-line-strings` defines whether to lint indentation [...]

### Default values

```yaml
indentation:
  spaces: consistent
  indent-sequences: true
  check-multi-line-strings: false
```

### Examples

1. `With indentation: {spaces: 1}`

   the following code snippet [...]

**Comment by adrienverge** [comment]
@bewuethr thanks for pointing this, a PR would indeed be welcome.

Maybe we could even add `rules:` to your example, so that new users have ready-to-paste snippets, and don't struggle with configuration?

---

### Default values
```yaml
rules:
  indentation:
    spaces: consistent
    indent-sequences: true
    check-multi-line-strings: false
```
